global_hub/global_hub_install_acm.adoc

```diff
@@ -1,19 +1,19 @@
 [#global-hub-install-acm]
-= Installing {global-hub} on an existing {acm-short} hub cluster (Technology Preview)
+= Installing {global-hub} on an existing {acm-short} hub cluster
 
-When you create the `MultiClusterGlobalHub` custom resource (CR), you must set the `installAgentOnLocal` field to `true`. The `installAgentOnLocal` field determines if you can deploy {global-hub}  on an existing  {acm-short} hub cluster.  
+You can install {global-hub} in an existing {acm-short} hub cluster, and enable the {global-hub} agent in the {acm-short}. 
+with this:
+- All the clusters, policies, and events in this {acm-short} will be send to kafka and then stored to globalhub database
+- Other components could easily consume the events in kafka, eg: integrate with `Red Hat Advanced Cluster Security` 
```

Comment:
make a link to point to https://github.com/stolostron/stolostron/tree/main/dev-preview#global-hub-integration-with-red-hat-advanced-cluster-security

Reply:
rephrase

- All the clusters, policies, and events in this {acm-short} will be sent to Kafka and then stored in the globalhub database.
- Since the events will be sent to Kafka, other components can easily consume the events in Kafka, e.g., [integration with Red Hat Advanced Cluster Security](https://github.com/stolostron/stolostron/tree/main/dev-preview#global-hub-integration-with-red-hat-advanced-cluster-security) or EDA integration.
- Since the data will be stored in the globalhub database, you can view cluster and policy information in Grafana dashboards.